src/Pagination/constants.js

```diff
@@ -1,2 +1,16 @@
-/* eslint-disable import/prefer-default-export */
 export const ELLIPSIS = '...';
+
+export const PAGINATION_VARIANTS = {
+  default: 'default',
+  secondary: 'secondary',
+  reduced: 'reduced',
+  minimal: 'minimal',
+};
+
+export const PAGINATION_BUTTON_LABEL_PREV = 'Previous';
```

Comment:
A good option is to move these constants related to PAGINATION_BUTTON to an object.
For example: `export const PAGINATION_BUTTON = { label: { prev: 'Previous' ... }, iconButton: { nextAlt: 'Go to next page', ... } }`

Reply:
I think in this case it's slightly better to separate these constants, just like they are right now. To me it seems that they are easier to use this way - instead of dragging the whole object across multiple files I can import just a single constant which is needed for the specific component.

I'd like to keep them as they are right now.

Reply:
Ok, thanks 👍